features/storage/FEATURE_STORAGE/TESTS/cfstore/create/create.cpp

```diff
@@ -75,61 +75,61 @@ UVISOR_BOX_CONFIG(cfstore_create_box1, UVISOR_BOX_STACK_SIZE);
 };
 
 /* table 2: to CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_01 grown to CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_02 */
 static cfstore_kv_data_t cfstore_create_test_01_data_step_02[] = {
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04,
-        { NULL, NULL},
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04,
+    { NULL, NULL},
 };
 
 /* table 3: to CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_02 shrunk to CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_03 */
 static cfstore_kv_data_t cfstore_create_test_01_data_step_03[] = {
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04,
-        { NULL, NULL},
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04,
+    { NULL, NULL},
 };
 
 /* table 3: CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_03 deleted */
 static cfstore_kv_data_t cfstore_create_test_01_data_step_04[] = {
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_03,
-        CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04,
-        { NULL, NULL},
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_HEAD_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_03,
+    CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04,
+    { NULL, NULL},
```

Comment:
Not sure what you require here. Astyle left this untouched. Changing to {NULL, NULL}. Hope this is as intended.